astrodb_utils/sources.py

```diff
@@ -462,3 +466,23 @@ def find_survey_name_in_simbad(sources, desig_prefix, source_id_index=None):
         )
 
     return result_table
+
+
+def strip_unicode_dashes(source):
```

Comment:
I like this approach. What about naming it more generally, such as string_formatter? This way you can use the same function for more than just dashes.